docs/ref/settings.txt

```diff
@@ -665,8 +665,22 @@ TEST
 Default: ``{}``
 
 A dictionary of settings for test databases; for more details about the
-creation and use of test databases, see :ref:`the-test-database`. The
-following entries are available:
+creation and use of test databases, see :ref:`the-test-database`.
+An example of a simple settings file containing info about the test database
```

Comment:
If this is meant to be a separate paragraph, it needs a newline above. If not, then you can flow "An example of" on the previous line.